src/azure-cli/azure/cli/command_modules/synapse/manual/operations/accesscontrol.py

```diff
@@ -141,20 +140,18 @@ def _resolve_object_id(cmd, assignee, fallback_to_object_id=False):
                            "Please using --assignee-object-id GUID to specify assignee accurately"
                            .format(assignee=assignee))
 
-        return result[0].object_id
-    except (CloudError, GraphErrorException):
+        return result[0]["id"]
+    except GraphError:
         if fallback_to_object_id and is_guid(assignee):
             return assignee
         raise
 
 
 def _get_object_stubs(graph_client, assignees):
-    from azure.graphrbac.models import GetObjectsParameters
     result = []
     assignees = list(assignees)  # callers could pass in a set
     for i in range(0, len(assignees), 1000):
-        params = GetObjectsParameters(include_directory_object_references=True, object_ids=assignees[i:i + 1000])
-        result += list(graph_client.objects.get_objects_by_object_ids(params))
+        result += list(graph_client.directory_object_get_by_ids(assignees[i:i + 1000]))
```

Comment:
Has this code been tested?

This is not how `directory_object_get_by_ids` is used. See https://github.com/Azure/azure-cli/blob/33cbd01f75c7d4c3814de5a1cbf6f42de7890467/src/azure-cli/azure/cli/command_modules/role/custom.py#L1573-L1586